packages/components/src/ProgressCircular/ProgessCircular.story.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+
+ MIT License
+
+ Copyright (c) 2020 Looker Data Sciences, Inc.
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+
+ */
+
+import React from 'react'
+import { Story } from '@storybook/react/types-6-0'
+import { ProgressCircular, ProgressCircularProps } from './ProgressCircular'
+
+const Template: Story<ProgressCircularProps> = (args) => (
+  <ProgressCircular {...args} />
+)
+
+export const Indeterminate = Template.bind({})
+Indeterminate.args = {
+  determinate: false,
+  progress: 0,
+  size: 'large',
+}
+
+export const Determinate = Template.bind({})
+Determinate.args = {
+  determinate: true,
+  progress: 0.5,
+  size: 'large',
+}
+
+export default {
+  component: ProgressCircular,
+  title: 'ProgressCircular',
+}
```

Comment:
I noticed that the default Storybook controls are oriented to whole numbers, which is definitely awkward here: 

![storybook-controls](https://user-images.githubusercontent.com/238293/103026937-c8c04c80-4509-11eb-9ea0-5845126edf6f.gif)
I just looked it up and luckily that can be customized to be a little more helpful 🥳 

```suggestion
export default {
  component: ProgressCircular,
  title: 'ProgressCircular',
  argTypes: {
    progress: {
      control: { type: 'range', min: 0, max: 1, step: 0.1 },
    },
  },
}
```

![slider-controls](https://user-images.githubusercontent.com/238293/103027131-22287b80-450a-11eb-814f-fd22fc26eccb.gif)

Reply:
🥳